build/build-image/cross/Dockerfile

```diff
@@ -38,3 +38,23 @@ RUN apt-get install -y g++ && apt-get clean && rm -rf /var/lib/apt/lists/* &&\
     rm -rf protobuf-3.0.0-beta-2 &&\
     protoc --version
 
+# (set an explicit GOARM of 5 for maximum compatibility)
+ENV GOARM 5
```

Comment:
As a minor optimization all of these can be folded into one env statement and save everyone who pulls a small bit of time (5-10s on devicemapper loop back)